packages/app/obojobo-document-engine/src/scripts/oboeditor/components/toolbars/format-menu.js

```diff
@@ -33,45 +35,55 @@ const FormatMenu = props => {
 		name: 'Paragraph styles',
 		type: 'sub-menu',
 		menu: [
-			{ 
-				name: 'Normal Text', 
-				type: 'action', 
-				action: () => props.editor.changeToType(TEXT_NODE) 
+			{
+				name: 'Text',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+Space',
+				shortcutMac: '⌘+Shift+Space',
+				action: () => props.editor.changeToType(TEXT_NODE)
 			},
-			{ 
-				name: 'Heading 1', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 1 }) 
+			{
+				name: 'Heading 1',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+1',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 1 })
 			},
-			{ 
-				name: 'Heading 2', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 2 })  
+			{
+				name: 'Heading 2',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+2',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 2 })
 			},
-			{ 
-				name: 'Heading 3', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 3 })  
+			{
+				name: 'Heading 3',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+3',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 3 })
 			},
-			{ 
-				name: 'Heading 4', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 4 })  
+			{
+				name: 'Heading 4',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+4',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 4 })
 			},
-			{ 
-				name: 'Heading 5', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 5 })  
+			{
+				name: 'Heading 5',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+5',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 5 })
 			},
-			{ 
-				name: 'Heading 6', 
-				type: 'action', 
-				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 6 })  
+			{
+				name: 'Heading 6',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+6',
+				action: () => props.editor.changeToType(HEADING_NODE, { headingLevel: 6 })
 			},
-			{ 
-				name: 'Code', 
-				type: 'action', 
-				action: () => props.editor.changeToType(CODE_NODE)  
+			{
+				name: 'Code',
+				type: 'action',
+				shortcut: 'Ctrl+Shift+C',
+				shortcutMac: '⌘+Shift+C',
```

Comment:
On the mac both cmd+shift+1 and ctrl+shift+1 work, however for this one only ctrl works. Is there an existing keyboard shortcut we're working around for this one?

Reply:
I added the shifted versions of the keys here, which should make CMD work - it always worked on my machine, though, so let me know if it doesn't work